### ISSUE TITLE
Choose starting portal

### ISSUE THREAD

**Issue by egor-n** [jpeterbaker/maxfield#18]
Would be nice to select the starting portal as an optional value.

**Comment by jpeterbaker** [comment]
I agree. I'd like to accept a start position in the input file and the agent-routing routine would know that they would start there. I'll try to get this in soon.
